engine/pmem_allocator/free_list.cpp

```diff
@@ -160,12 +161,6 @@ void FreeList::Push(const SizedSpaceEntry &entry) {
     if (thread_cache.active_entries[entry.size].size() >= kMaxCacheEntries) {
       std::lock_guard<SpinMutex> lg(thread_cache.spins[entry.size]);
       thread_cache.backup_entries[entry.size].emplace_back(entry.space_entry);
```

Comment:
`PMEMAllocator` and `FreeList` both have a inner class `ThreadCache`. That's quite confusing, especially when they functions totally differently but share similar name.

Reply:
I think it's ok, they both indicate a thread local cache, and since they are in different context, the role is clear.